### DISCUSSION TITLE
Is it possible to get server side data for +error.svelte?

### DISCUSSION THREAD

**Discussion by korywka** [sveltejs/kit#10591] [Q&A] [answered]
Something like we do for pages, but `+error.server.js`.

**Comment by dennisregalado** [comment]
definitely see a use case for this, currently using a hacky workaround to render a 404 page with data from Sanity with the correct error code response

**Reply by benmccann** [reply]
The status code being wrong would probably be fixed by https://github.com/sveltejs/kit/pull/10565

**Comment by korywka** [comment]
@benmccann should we create an issue for this feature request or leave it as discussion?

**Reply by benmccann** [reply]
Can you share some details on the use case?

**Reply by korywka** [reply]
When user gets 404, but you want to suggest similar pages. For example this one: https://macosbin.com/bin/sofari
If you do it with CSR, you need to expose to public GET route for XHR.

**Reply by korywka** [reply]
Also may be helpful to generate unique error ID that can help to resolve problem, like CF do:

![image](https://github.com/sveltejs/kit/assets/988471/75d61d8e-e340-4b3c-88c1-68a0e53405b6)

**Reply by teemingc** [reply, marked as answer]
an issue for this exists here https://github.com/sveltejs/kit/issues/6106
